app/main/browser-window.js

```diff
@@ -52,8 +61,13 @@ export async function createBrowserWindow(userAgentClient, onload) {
     if (onload) {
       onload();
     }
+    // The client needs to know where the UA service is in order to connect
```

Comment:
We might be able to avoid the round trip by having the UI fish host and service from it's initial URL, or from query parameters.  (I did this once before; it worked fine.)  Your choice.